src/Components/StudentsList/Pagination.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState } from "react";
```

Comment:
Our convention is to use styled-components. Consider using this. The entire component should be styled according to the mockup.